package.json

```diff
@@ -84,8 +84,13 @@
     "playwright": "^1.40.1",
     "prettier": "^3.0.0",
     "prettier-plugin-glsl": ">=0.1.2 <1",
+    "prettier-plugin-ini": "^1.1.0",
+    "prettier-plugin-latex": "^2.0.1",
     "prettier-plugin-organize-imports": "^3.2.4",
+    "prettier-plugin-properties": ">=0.3.0 <1",
+    "prettier-plugin-sh": ">=0.13.0 <1",
```

Comment:
Another thought, they're all `devDependencies`, and we're using lock file, does they actually work as expected?

Reply:
Ah, I know what you mean.

Although they're no longer peer dependencies and won't trigger any version mismatching alert, I still think we better specify the supported version ranges explicitly, so at least we can get a hint about which versions are supported by this plugin. What do you think?

Reply:
> so at least we can get a hint about which versions are supported by this plugin

I'm not quite sure to understand this, and the range makes difficult to upgrade plugins.

For example, current `>=0.1 <1` and I want to try `v0.2`, how can I upgrade in this project?

Reply:
> npm i xxx@0.2 should be good? [0.2 satisfies >=0.1 <1](https://jubianchi.github.io/semver-check/#/%3E%3D0.1%20%3C1/0.2).

Would this command change the field in `package.json`? Or would it only change `package-lock.json`? That's not clear enough.

You can see `0.x` and `0.(x+1)` as two major/incompatible versions before `v1.0`. As I said, you didn't test `0.(x+1)` at all before it's been released.

Reply:
> Would this command change the field in `package.json`? Or would it only change `package-lock.json`? That's not clear enough.

You're right, this will change the field in `package.json` and `package-lock.json`. I'll have to reset the version range in `package.json`.

> As I said, you didn't test 0.(x+1) at all before it's been released.

Strictly speaking, yes, we shouldn't relax the version range too much. `^0.2.0` seems to be the better choice if `0.3.0` hasn't been published and tested. Yet there're still cases where `0.1.x` and `0.2.x` are both published and supported by this plugin, in this case `^0.2.0` can not reflect this plugin supports `0.1.x` versions. We'll have to use sth like `>=0.1 <0.3`.

I don't know, maybe we just don't track the minimal support versions and use `^` everywhere?

Reply:
Yep this is my initial thought: https://github.com/Sec-ant/prettier-plugin-embed/pull/40#discussion_r1413482340, but I'm convinced we should change these version ranges.

I wish npm could provide a kind of dependencies just works like the peer deps before npm 7, sigh.

Reply:
Just rebased and fixed all version specifiers.